content/projects/knowtox.md

```diff
@@ -0,0 +1,59 @@
+---
+title: "KnowTox"
+date: 2020-05-20T16:27:08+02:00
+
+weight: 10
+nav: false
+
+people:
+- key: andrea-lilian.morger
+- key: andrea.volkamer
+
+collaborators:
+- name: Miriam Mathea
+  link: https://www.basf.com/
+- name: Janosch Achenbach
+  link: https://www.basf.com/
+- name: Robert Landsiedel
+  link: https://www.basf.com/
+- name: Roland Buesen
+  link: https://www.basf.com/
+- name: Antje Wolf
+  link: https://www.basf.com/
+- name: Klaus-Juergen Schleifer
+  link: https://www.basf.com/
+
+
+publications:
+- morger_2020_knowtox
+
+external_resources:
+- name: knowtox_manuscript_SI
+  link: https://github.com/volkamerlab/knowtox_manuscript_SI
+  icon: github
+  more: SI notebook - build & evaluate conformal predictor and apply to external data
+
+funding:
+- name: Bundesministerium für Bildung und Forschung, grant ID 031A262C
+- name: HaVo-Stiftung
+- name: We thank BASF for supporting this project by providing financial resources
+
+summary: |
+  KnowTox is a novel pipeline that combines three different _in silico_ toxicology approaches to allow for confident prediction of potentially toxic effects of novel chemical entities, i.e. machine learning models, alerts for toxic substructures and computational support for read-across.
+---
+
+Computational tools for toxicity prediction are promising in the process of reducing, refining and replacing animal testing. To assess the toxicity of novel chemical entities, regulatory agencies require in vivo testing for several toxic endpoints. In 2010, roughly 2.9 million laboratory animals have been deployed in Germany, with an increase of 6% since 2008. Thus, the establishment of alternative methods, and with it the reduction of animal testing, is of utmost importance.
+
+KnowTox is a novel pipeline that combines three different _in silico_ toxicology approaches to allow for confident prediction of potentially toxic effects of novel chemical entities, i.e. machine learning models, alerts for toxic substructures and computational support for read-across.
```

Comment:
```suggestion
While grouping of molecules and read-across are already the most widely used non-animal methods (ECHA), *KnowTox* is a novel pipeline that combines three different _in silico_ toxicology approaches to substantiate the forecasts and to allow for confident prediction of potentially toxic effects of novel chemical entities. KnowTox combines machine learning models, alerts for toxic substructures and computational support for read-across.
```
add external link to 'ECHA': https://echa.europa.eu/-/alternatives-to-animal-testing-continue-to-be-widely-used

Reply:
if we agree on this new sentence, it must also be adapted in the 'summary' part above.

Reply:
and please adapt the funding sentence to:
We thank BASF for financial support.
(instead of We thank BASF for supporting this project by providing financial resources)

Reply:
I am not sure if KnowTox is really able to substantiate the forecasts. Is it possible that RL meant, that it would be helpful for KnowTox if the forecasts (made by KnowTox) could be substantiated (e.g. with experimental data)?
If @AndreaVolkamer is certain about it, I agree to to the new version. Otherwise, I would suggest to shorten the first sentence to `While grouping of molecules and read-across are already the most widely used non-animal methods (ECHA), *KnowTox* is a novel pipeline that combines three different _in silico_ toxicology approaches to allow for confident prediction of potentially toxic effects of novel chemical entities.`